### ISSUE TITLE
Keyboard speed control

### ISSUE THREAD

**Issue by scanlime** [Chocobozzz/PeerTube#966]
This is a minor enhancement really, but it's something I use constantly on youtube and I miss here.

">" to increase speed, "<" to decrease

**Comment by BO41** [comment]
could someone drop some information about how to solve this issue?
I don't know where to start and how to solve it.

I found `angular2-hotkeys` as a dependency and where it is used, but I don't know where to set the hotkeys for the video.
And what can video.js do? 

It should be simple:
- set the hokeys
- if pressed, select the next speed option if availabe

**Comment by rigelk** [comment]
@BO41 `angular2-hotkeys` is more general. It is used to define application-wide hotkeys, while `videojs-hotkeys` is used for those within videojs (which already has support for a few basic hotkeys). See: https://github.com/ctd1500/videojs-hotkeys

We define `videojs-hotkeys` hotkeys here: https://github.com/Chocobozzz/PeerTube/blob/81cdf382ffab82357320b575533c784878d4b4c1/client/src/assets/player/peertube-player.ts#L70-L72

**Comment by rigelk** [comment]
Something along the lines of:

```js
rewindKey: function(event, player) {
  // override rewind to trigger when pressing the left arrow (37) or < (50) keys
  return ((event.which === 37) || (event.which === 50));
}
```

You can get keycodes via http://keycode.info/ for instance.

**Comment by rigelk** [comment]
@BO41 have you started working on the code? Because I have something close to and can take the issue if you don't mind.

**Comment by rigelk** [comment]
Then consider it implemented as of 52b1ba03 ;)
